zfs.h

```diff
@@ -122,7 +122,7 @@ int dataset_clone(dataset_list_ptr dataset, const char *target, nvlist_ptr props
 int dataset_snapshot(const char *path, boolean_t recur, nvlist_ptr props);
 int dataset_rollback(dataset_list_ptr dataset, dataset_list_ptr snapshot, boolean_t force);
 int dataset_promote(dataset_list_ptr dataset);
-int dataset_rename(dataset_list_ptr dataset, const char* new_name, boolean_t recur, boolean_t force_unm);
+int dataset_rename(dataset_list_ptr dataset, const char* new_name, boolean_t recur, boolean_t no_unm, boolean_t force_unm);
```

Comment:
not sure if this is ok, or if it best to mimick the new C api, and actually create renameflags_t and expose it.

Reply:
note sure if this is still maintained.
if we have to fork anyway, what about just removing the broken interfaces?